Lib/idlelib/tree.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, canvas, parent, item):
         self.selected = False
         self.children = []
         self.x = self.y = None
+        self.dy = 0
```

Comment:
There is a new instance for every displayed line, so this results in reduntantly recalculating dy for every line.  The time is not really a problem, but it is misleading.  Make dy a class variable.  Still read as 'self.dy', but set as 'Tree.dy'.